### ISSUE TITLE
Wrong string for pt-BR locale

### ISSUE THREAD

**Issue by lcneves** [airbnb/polyglot.js#139]
index.js, line 102:
`french: ['fr', 'tl', 'pt-br'],`

Should be:
`french: ['fr', 'tl', 'pt-BR'],`

**Comment by ljharb** [comment]
cc @schleyfox to confirm that we don't need to keep both capitalizations

**Comment by schleyfox** [comment]
I don't think we do any handling of the locales that are passed in that would make this case insensitive, so changing the capitalization could be breaking. You'd probably need both capitalizations.
